src/components/CharactersPage/CharacterForm/CharacterForm.js

```diff
@@ -71,9 +80,45 @@ const CharacterForm = ({ onClose }) => {
     reValidateMode: "onSubmit",
   })
 
+  useEffect(() => {
+    if (!character) return
+
+    const {
+      name,
+      eyeColor,
+      height,
+      gender,
+      birthYear,
+      homeworld,
+      species,
+      filmConnection,
+    } = character
+    reset({
+      name: name || "",
+      eyeColor: eyeColor || "",
```

Comment:
You don't need to be providing defaults for fields that are required, only the optional ones